src/viperleed/calc/lib/fortran_utils.py

```diff
@@ -5,17 +5,64 @@
 
 __authors__ = (
     'Florian Kraushofer (@fkraushofer)',
+    'Alexander M. Imre (@amimre)',
     'Michele Riva (@michele-riva)',
     )
 __copyright__ = 'Copyright (c) 2019-2024 ViPErLEED developers'
 __created__ = '2024-08-26'
 __license__ = 'GPLv3+'
 
+import re
+import subprocess
+import shutil
```

Comment:
swap for alphabetic order:
```suggestion
import shutil
import subprocess
```